### PR TITLE
Small updates for TensorFlow 1.0 compatibility.

### PR DESCRIPTION
**Pull request by dyelax**

### CODE REVIEW

**Comment by rajarsheem** [comment]
@dyelax : Thanks for the PR! However few TODOs are still left. You need to change the tf version in the travis_install.sh file. Currently it's 0.7 everywhere. This is needed to pass the travis build tests.

Also, the tf version in readme needs to be changed.

Thanks again. Waiting for your updated PR !

**Comment by dyelax** [comment]
@rajarsheem Should be fixed now! Going to submit another pull request soon. Working on a function for "finetuning" the entire autoencoder after pretraining each layer

File: deepautoencoder/stacked_autoencoder.py [hunk 1]
```diff
@@ -127,10 +140,47 @@ def run(self, data_x, data_x_, hidden_dim, activation, loss, lr,
                 print('epoch {0}: global loss = {1}'.format(i, l))
         # debug
         # print('Decoded', sess.run(decoded, feed_dict={x: self.data_x_})[0])
-        self.weights.append(sess.run(encode['weights']))
-        self.biases.append(sess.run(encode['biases']))
+
```
Comment (rajarsheem) [inline deepautoencoder/stacked_autoencoder.py:143]: This breaks the code inside transform(). Because you are appending tf.Variable object to self.weights but transform() assumes that the weights are in a numpy array. You may want to somehow modify transform() to operate on tf.Variable by running a session.

File: deepautoencoder/stacked_autoencoder.py [hunk 2]
```diff
@@ -76,6 +82,13 @@ def fit(self, x):
                              batch_size=self.batch_size,
                              lr=self.lr, print_step=self.print_step)
 
+        if finetune:
```
Comment (rajarsheem) [inline deepautoencoder/stacked_autoencoder.py:85]: AFAIK, you conduct fine tuning after the whole network is trained and not after training of each layer.

File: deepautoencoder/stacked_autoencoder.py [hunk 3]
```diff
@@ -127,10 +140,47 @@ def run(self, data_x, data_x_, hidden_dim, activation, loss, lr,
...
         # debug
         # print('Decoded', sess.run(decoded, feed_dict={x: self.data_x_})[0])
-        self.weights.append(sess.run(encode['weights']))
-        self.biases.append(sess.run(encode['biases']))
+
+        self.weights.append(encode['weights'])
+        self.biases.append(encode['biases'])
         return sess.run(encoded, feed_dict={x: data_x_})
 
+    def finetune(self, data_x, data_x_, loss, lr, print_step, epoch,
```
Comment (rajarsheem) [inline deepautoencoder/stacked_autoencoder.py:148]: Fine tuning is a supervised process, isn't it? I mean that you take in the labels (train_Y from test.py) and perform a softmax classification in the last layer of the autoencoder and update (or fine-tune) the weights of the whole autoencoder ! :smiley:
